src/main/kotlin/com/autonomousapps/DependencyAnalysisExtension.kt

```diff
@@ -63,6 +63,8 @@ open class DependencyAnalysisExtension(objects: ObjectFactory) {
   fun issues(action: Action<IssueHandler>) {
     action.execute(issueHandler)
   }
+
+  var dependencyRenamingMap: Map<String, String>? = null
```

Comment:
We should avoid nullable vars. Instead, use gradle's `MapProperty`. See https://docs.gradle.org/current/userguide/lazy_configuration.html#working_with_maps. Always prefer the Property/Provider API when possible.

Reply:
I didn't know about the map property. Thx.
For the null, we could use an empty map instead, but the performance could be a little degraded. It should work though.

Reply:
Can this also be a `MapProperty`?